src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorCSharpLanguageMiddleware.ts

```diff
@@ -48,12 +48,14 @@ export class RazorCSharpLanguageMiddleware implements LanguageMiddleware {
 
             // Re-map each edit to its position in the original Razor document.
             for (const edit of edits) {
-                const remappedResponse = await this.serviceClient.mapToDocumentRange(
+                const remappedResponse = await this.serviceClient.mapToDocumentRanges(
```

Comment:
This will need to be cleaned up as part of https://github.com/dotnet/aspnetcore/issues/22449